README.MD

```diff
@@ -11,9 +11,14 @@ environment for the following projects:
 
 ## Design
 
-The setup includes one Apache server acting as reverse proxy, 1 REST WHOIS
-server, 1 registry instance serving EPP requests and 1 registry server serving
-all portals interface. Ruby projects are running Puma servers.
+The setup includes:
+  * 1 Apache server acting as reverse proxy
+  * 1 registry instance serving EPP requests
+  * 1 registry server serving all portals interface.
+  * 1 auction server serving auction interface
+  * 1 auction server running background (delayed) tasks
+  * 1 REST WHOIS server
+Ruby projects are running Puma servers.
```

Comment:
Only Rest whois? the docs seem to mention whois on port 43 from what i conclude that whoisd is included aswell

Reply:
Actual scope of this PR was to get registry & auction sites up and running. I haven't looked into WHOIS part nor it's configuration yet, in fact I haven't even tried yet to deploy them locally. True, this part might be faulty.